schema/CVE_Record_Format.json

```diff
@@ -500,6 +500,93 @@
             "required": ["orgId"],
             "additionalProperties": false
         },
+        "applicability": {
+            "type": "array",
+            "items": {
+                "$ref": "#/definitions/applicabilityElement"
+            }
+        },
+        "applicabilityElement": {
+            "description": "Affected products defined using one or more software identifiers and applicability parameters. An operator property allows AND or OR logic between identifiers or combinations thereof. The negate and vulnerable Boolean properties allow applicability combinations to be inverted and/or defined as vulnerable or not. Multiple version fields are provided for capturing ranges of products when defining a Match String Range. NOTE: When defining an applicability block, it is recommended that it align with (as much as possible) the product data provided within the affected block.",
+            "properties": {
+                "operator": {
+                    "type": "string",
+                    "enum": ["AND", "OR"]
+                },
+                "negate": {
+                    "type": "boolean"
+                },
+                "nodes": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/node"
+                    }
+                }
+            }
+        },
+        "node": {
+            "description": "Defines a configuration node in an applicability statement.",
+            "properties": {
+                "operator": {
+                    "type": "string",
+                    "enum": ["AND", "OR"]
+                },
+                "negate": {
+                    "type": "boolean"
+                },
+                "cpeMatch": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/cpe_match"
+                    }
+                },
+                "purlMatch": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/purl_match"
+                    }
+                }
+            },
+            "allOf": [
+                { "required": ["operator"] },
+                {
+                    "anyOf": [
+                        { "required": ["cpeMatch"] },
+                        { "required": ["purlMatch"] }
+                    ]
+                }
+            ]
+        },
+        "purl_match": {
+            "description": "purl match string or range",
```

Comment:
What do you mean by purl_match?

Reply:
`purl_match` is the internal name of the object type as it exists in the schema, it is not a user-facing name, and is named thus to stylistically match the equivalent existing `cpe_match` structure.